src/Alert/index.js

```diff
@@ -0,0 +1,151 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+import { withTheme } from "theming";
+
+import {
+  InformationCircle,
+  Check,
+  Alert as AlertTriangle,
+  AlertCircle,
+  CloseCircle,
+} from "../icons";
+import { Button } from "../index";
+
+const Icon = ({ icon, type }) => {
+  let renderIcon;
+  if (typeof icon === "boolean") {
+    if (type === "info") {
+      renderIcon = <InformationCircle />;
+    }
+    if (type === "success") {
+      renderIcon = <Check />;
+    }
+    if (type === "warning") {
+      renderIcon = <AlertTriangle />;
+    }
+    if (type === "critical") {
+      renderIcon = <AlertCircle />;
+    }
+  } else {
+    return icon;
+  }
+  return renderIcon;
+};
+
+type Props = {
+  type: "info" | "success" | "warning" | "critical",
+  title?: string,
```

Comment:
title could be string or boolean if user pass

```
title={false}
```

Reply:
@tomashapl plz explain your suggest more, I don't think there is no need to define title as boolean also